Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -197,6 +197,21 @@ internal void SetClientOwner(NetworkConnection conn)
         /// </summary>
         public static void ResetNextNetworkId() => nextNetworkId = 1;
 
+        /// <summary>
+        /// The delegate type for the clientAuthorityCallback.
+        /// </summary>
+        /// <param name="conn">The network connection that is gaining or losing authority.</param>
+        /// <param name="identity">The object whose client authority status is being changed.</param>
+        /// <param name="authorityState">The new state of client authority of the object for the connection.</param>
+        public delegate void ClientAuthorityCallback(NetworkConnection conn, NetworkIdentity identity, bool authorityState);
+
+        /// <summary>
+        /// A callback that can be populated to be notified when the client-authority state of objects changes.
+        /// <para>Whenever an object is spawned using SpawnWithClientAuthority, or the client authority status of an object is changed with AssignClientAuthority or RemoveClientAuthority, then this callback will be invoked.</para>
+        /// <para>This callback is only invoked on the server.</para>
+        /// </summary>
+        public static ClientAuthorityCallback clientAuthorityCallback;
```

Comment:
This was obsolete before it was removed.  It should stay obsolete.

There is really no good reason to have this other than backwards compatibility.   The server has to explicitly call AssignAuthority,  so there is no need to tell the server about what it just did.

Reply:
No, it has a legitimate use case, and having it obsolete is how it got removed.